src/common/vfs.h

```diff
@@ -62,6 +62,11 @@ struct OCSYNC_EXPORT VfsSetupParams
     QString providerName;
     QString providerVersion;
 
+    /** when registering with the system we might use
+     *  a different presentaton to identify the accounts
+     */
+    bool multipleAccountsKnow = false;
```

Comment:
Strange var name

Reply:
multipleAccountsKnown... thats we I ask to review my code 🙈

Reply:
> 
> 
> This needs restarting after adding an account for the display name to update for the old account, right?

Usually only a restart of the client, as the registration is updated every time.

Reply:
Yes, that's what I meant